kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/ProxyReconcilerIT.java

```diff
@@ -93,24 +120,34 @@ void testDelete() {
         var cr = doCreate();
         extension.delete(cr);
 
-        await().untilAsserted(() -> {
+        await().alias("Secret was deleted").untilAsserted(() -> {
             var secret = extension.get(Secret.class, ProxyConfigSecret.secretName(cr));
             assertThat(secret).isNull();
-
+        });
+        await().alias("Deployment was deleted").untilAsserted(() -> {
             var deployment = extension.get(Deployment.class, ProxyDeployment.deploymentName(cr));
             assertThat(deployment).isNull();
-
-            var service = extension.get(Service.class, ProxyService.serviceName(cr));
-            assertThat(service).isNull();
         });
+        await().alias("Services were deleted").untilAsserted(() -> {
+            for (var cluster : cr.getSpec().getClusters()) {
+                var service = extension.get(Service.class, ClusterService.serviceName(cluster));
+                assertThat(service).isNull();
+            }
+        });
+        LOGGER.atInfo().log("Test finished");
```

Comment:
```suggestion
```

Reply:
This is actually useful to distinguish between exceptions that happen during the test from those which happen when the Kube resources are being torn down (e.g. namespace deleted).

Reply:
urghhhh. Ok leave it in